utils/mvn/utils.go

```diff
@@ -1,152 +1,49 @@
 package mvnutils
 
```

Comment:
It looks like this file includes that unused code. For example - 
```
func (config *mvnRunConfig) runCmd() error {
	command := config.GetCmd()
	command.Stderr = os.Stderr
	command.Stdout = os.Stderr
	return coreutils.ConvertExitCodeError(errorutils.CheckError(command.Run()))
}
```
Let's go ver the code and check what should be removed.